opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -118,6 +118,23 @@ def add_event(
         timestamp if the `timestamp` argument is omitted.
         """
 
+    @abc.abstractmethod
+    def add_link(
+        self,
+        context: "SpanContext",
+        attributes: types.Attributes = None,
+    ) -> None:
+        """Adds a `Link`.
+
+        Adds a single `Link` with the `SpanContext` of the span to link to and,
+        optionally, attributes passed as arguments. Implementations may ignore
+        calls with an invalid span context.
```

Comment:
```suggestion
```

Reply:
Any reason behind removing this documentation? I've followed the documentation from the `add_event()` method above.

Reply:
It seemed like a copy paste from the spec. On a second thought, this is fine.